isminet/models/base.py

```
@@ -1,19 +1,33 @@
 T = TypeVar("T")
 
 
 class UnifiBaseModel(BaseModel):
-    """Base model with common configuration for all UniFi models."""
+    """Base model for all UniFi Network API models."""
 
     model_config = ConfigDict(
-        str_strip_whitespace=True,
-        str_min_length=1,
-        strict=True,
+        extra="ignore", str_strip_whitespace=True, validate_assignment=True
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)